### PR TITLE
Fix editor2 for aap

### PR DESCRIPTION
**Pull request by petrjasek**
- fix find&replace
- add option to disable toolbar
- add option to disable embeds

also fixed a memory leak in add-content

~~initially I've put some common parts for editor1 and editor2 into editor/utils.js, but not sure if it makes sense if we dump editor1~~

### CODE REVIEW

File: app/index.html [hunk 1]
```diff
@@ -123,6 +123,7 @@
     <script src="scripts/superdesk/directives/sdWithParams.js"></script>
 
     <script src="scripts/superdesk/editor/editor.js"></script>
+    <script src="scripts/superdesk/editor/utils.js"></script>
```
Comment (akintolga) [inline app/index.html]: Is `utils.js` missing from the PR?

Comment (petrjasek) [inline app/index.html]: I didn't use it at the end, will remove this

**Comment by petrjasek** [comment]
ping @superdesk/developers

**Comment by mdhaman** [comment]
:+1:

**Comment by akintolga** [comment]
:+1:

**Pull request merged**